config/api.yml

```diff
@@ -233,7 +233,7 @@
     :options:
     - :collection
     - :subcollection
-    :verbs: *gpd
+    :verbs: *gp
```

Comment:
Why is the DELETE action removed on resource.  You can add that back.  We support DELETE on resources that are zapped async, i.e. /api/providers,  if user sends a DELETE HTTP action, it's for fire and forget, they won't care about the task_id.  If they care, they'd simply use the delete POST action and leverage our action result signature.